packages/workspace/src/generators/new/generate-preset.ts

```diff
@@ -147,12 +147,14 @@ function getPresetDependencies({
       return { dependencies: {}, dev: { '@nrwl/node': nxVersion } };
 
     default: {
+      presetVersion =
```

Comment:
This was handled in here: https://github.com/nrwl/nx/commit/dc98485834611f14cc8abb2cce6d851e0e27fc4e